src/test/unit_tests/jest_tests/test_nc_nsfs_anonymous_cli.test.js

```diff
@@ -123,6 +123,58 @@ describe('manage nsfs cli anonymous account flow', () => {
             const resp = await exec_manage_cli(type, action, account_options);
             expect(JSON.parse(resp.stdout).error.message).toBe(ManageCLIError.InvalidArgument.message);
         });
+
+        it('cli create anonymous account with supplemental_groups - string', async () => {
+            let action = ACTIONS.DELETE;
+            const { type, anonymous, uid, gid } = defaults;
+            let account_options = { anonymous, config_root };
+            const resp = await exec_manage_cli(type, action, account_options);
+            const res_json = JSON.parse(resp.trim());
+            expect(res_json.response.code).toBe(ManageCLIResponse.AccountDeleted.code);
+
+            action = ACTIONS.ADD;
+            const supplemental_groups = "0,202,101";
```

Comment:
Please change single quotes to double quotes.
```suggestion
            const supplemental_groups = '0,202,101';
```